packages/core/src/codewhisperer/models/constants.ts

```diff
@@ -637,6 +637,9 @@ export const macJava8HomeHelpChatMessage =
 export const macJava11HomeHelpChatMessage =
     'To find the JDK path, run the following command in a new terminal:  `/usr/libexec/java_home -v 11`'
 
+export const macJava17HomeHelpChatMessage =
+    'To find the JDK path, run the following command in a new terminal:  `/usr/libexec/java_home -v 17`'
```

Comment:
Looks like the same string is duplicated 3 times, with only a 2-3 character difference. Can you use a templated string instead and pass the version number?

Reply:
@ntarakad-aws example of this: https://github.com/aws/aws-toolkit-vscode/blob/e8d62f1576a255ea105c4b7ba0b17b54cb9a87cb/packages/core/src/codewhisperer/models/constants.ts#L496